CMakeLists.txt

```diff
@@ -489,7 +492,7 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
   # The MoE kernel cutlass_moe_mm requires CUDA 12.3 or later (and only works
   # on Hopper). get_cutlass_moe_mm_data should only be compiled if it's possible
   # to compile MoE kernels that use its output.
-  cuda_archs_loose_intersection(SCALED_MM_ARCHS "9.0a;" "${CUDA_ARCHS}")
+  cuda_archs_loose_intersection(SCALED_MM_ARCHS "9.0a;10.0a" "${CUDA_ARCHS}")
   if(${CMAKE_CUDA_COMPILER_VERSION} VERSION_GREATER_EQUAL 12.3 AND SCALED_MM_ARCHS)
```

Comment:
I think we need to modify this so that the 10.0a kernels only gets compiled on 12.8 and up - right now I think this will break the build for CUDA 12.3 through 12.6

Reply:
The build for 12.4 is passing and tests are running. The SCALED_MM_ARCHS changes based on platform, correct? We take an intersection here - I believe that's why it is working as is

Reply:
OK, you're right, sorry! -- I had though we'd upgraded `docker-build-image` to use 12.8

Reply:
Thanks!